mobile/src/bundles/common/components/slider/constants/constants.ts

```diff
@@ -0,0 +1,7 @@
+const sliderValues = {
```

Comment:
```suggestion
const SLIDER_VALUES = {
```

Reply:
What about [B29](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md)?
Maybe it'd be better if i name it SliderValues and make names there in uppercase?

Reply:
ok, then rename file as enum